benchmark/fs/bench-stream-via-pipeline.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common.js');
+const tmpdir = require('../../test/common/tmpdir');
+const path = require('path');
+const fsp = require('fs/promises');
+const fs = require('fs');
+const stream = require('stream');
+
+const KB = 2 ** 10;
+const MB = 2 ** 20;
```

Comment:
nit: technically those are KiB and MiB, not that it matters ^^
```suggestion
const KiB = 2 ** 10;
const MiB = 2 ** 20;
```